spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -0,0 +1,212 @@
+[[dashboard]]
+= Dashboard
+
+[partintro]
+--
+This section describe how to use the Dashboard of Spring Cloud Data Flow.
+--
+
+[[dashboard-introduction]]
+== Introduction
+
+Spring Cloud Data Flow provides a browser-based GUI which currently has 6 sections:
+
+* **Apps** Lists all available applications and provides the control to register/unregister them
+* **Runtime** Provides the Dataflow cluster view with the list of all running applications
+* **Streams** Deploy/undeploy Stream Definitions
+* **Tasks** List, create, launch and destroy Task Definitions
+* **Jobs** Perform Batch Job related tasks
+* **Analytics** Create data visualizations for the various analytics modules
+
+Upon starting Spring Cloud Data Flow, the Dashboard is available at:
+
+"http://<adminHost>:<adminPort>/admin-ui"
+
+For example: http://localhost:9393/admin-ui
+
+If you have enabled https, then it will be located at https://localhost:9393/admin-ui
+
+If you have enabled security, a login form is available at http://localhost:9393/admin-ui/#/login
+
+**Note**: The default Dashboard server port is `9393`
+
+.The Spring Cloud Data Flow Dashboard
+image::images/dataflow-dashboard-about.png[The Spring Cloud Data Flow Dashboard, width=500]
```

Comment:
I may have made a mistake here.  But images are not showing up in my view of the docs.  I used the following build command `./mvnw clean package -DskipTests -P full -am`   is this correct?